.github/workflows/cd-syft-dev.yml

```diff
@@ -150,3 +162,24 @@ jobs:
           add: "."
           push: "origin main"
           cwd: "./infrastructure/"
+
+      - name: Cleanup Azure Container Registry
+        uses: azure/CLI@v1
+        with:
+          # SKIP_LINES = latest version dev & dev-<sha> (2 lines) + keep "n" previous version (n lines) + 1
+          inlineScript: |
+            ACR_REGISTRY_NAME=${{ secrets.ACR_REGISTRY_NAME }}
+            REPO_LIST=$(az acr repository list -n $ACR_REGISTRY_NAME -o tsv)
+
+            KEEP_PREV_VERSIONS=1
+            TAIL_FROM_LINE=$((2 + $KEEP_PREV_VERSIONS + 1))
+
+            for repo in $REPO_LIST
+            do
+                echo "Cleaning up '$repo'"
+                az acr repository show-tags --name $ACR_REGISTRY_NAME --repository $repo --orderby time_desc --output tsv | tail -n +$TAIL_FROM_LINE | xargs -r -I% az acr repository delete --name $ACR_REGISTRY_NAME --image $repo:% --yes
+            done
+
+      - name: Save Commit SHA
```

Comment:
Q: Should we use the actions/cache to save the cache key.I think this would save it locally in runner git files.